dipy/workflows/tests/test_denoise.py

```diff
@@ -0,0 +1,114 @@
+import os
+import numpy as np
+import numpy.testing as npt
+from dipy.testing import (assert_true, assert_false, assert_greater,
```

Comment:
why not use standard `unitests` package?

Reply:
we are using `numpy.testing`  as you can see above this line, but these functions are missing so we added them to `dipy.testing` to extend `numpy.testing`

Concerning `unitests` package,  I do not see any `assert_greater`, `assert_greater_equal`, `assert_less`, etc....

Reply:
have look at https://docs.python.org/3.6/library/unittest.html
`unittest.TestCase` has almost all asserts you can imagine... :-D

Reply:
ok, I see, thanks @Borda. I will fix that and create a new PR to update all DIPY later.

Reply:
After a quick test, it is really not convenient. I will keep it as it is. As you can see below, I find this syntax a bit heavy and I do want to have a global `TestCase` class instantiation. 

```
>>> from unittest import TestCase
>>> TestCase().assertTrue(1)
```

Reply:
I think that the good way of using `TestCase` is inheriting test class and call asserts as `self.`